facts.html

```diff
@@ -0,0 +1,6 @@
+<!doctype html>
+<html>
+  <body>
+    <h1>Hello, world!</h1>
```

Comment:
Update the contents of the h1 tag to be something more relevant to a facts page... like "Facts!" instead of "Hello, world!"

There is nothing actually "wrong" with your code, but I am suggesting changes on your PR so you know what it looks like.